sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Constants.java

```diff
@@ -198,7 +198,7 @@
         public static final String LOGGING_HANDLER_NAME = "logging-handler";
 
         // encryption
-        public static final String ENCRYPTION_ALGORITHM_ID = "encryptionAlgorithmId";
+        public static final String ENCRYPTION_ALGORITHM = "encryptionAlgorithm";
```

Comment:
Have have test before and after this change ?

Reply:
Yes, I have tested before and after this change. With **encryptionAlgorithmId** we are getting a null pointer exception now.